app/models/book.py

```diff
@@ -0,0 +1,11 @@
+class Book:
```

Comment:
A side effect of moving this to the models folder and importing it into routes is that when we convert this to be backed by SqlAlchemy, technically, we won't need to import the model file into `__init__.py`. There will already be an include path through which the Model-derived type will have been made visible to Alembic.

I added a comment to the work item for part 3 to keep an eye out for some of the implications.